pkgs/by-name/ch/choose-gui/package.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  xcbuild,
+  coreutils,
+  darwin,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "choose-gui";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "chipsenkbeil";
+    repo = "choose";
+    rev = version;
+    hash = "sha256-oR0GgMinKcBHaZWdE7O+mdbiLKKjkweECKbi80bjW+c=";
+  };
+
+  buildInputs = [
+    coreutils
+    darwin.apple_sdk.frameworks.Cocoa
+    xcbuild
+  ];
+
+  buildPhase =
+    let
+      arch =
+        with stdenv.hostPlatform;
+        if isx86_64 then
+          "x86_64"
+        else if isAarch64 then
+          "arm64"
+        else
+          throw "${pname}-${version} does not support ${parsed.cpu.arch} architecture";
+    in
+    ''
+      runHook preBuild
+      xcodebuild -arch ${arch} -configuration Release SYMROOT="./output" build
```

```suggestion
      xcodebuild -arch ${stdenv.hostPlatform.darwinArch} -configuration Release SYMROOT="./output" build
```

And drop `let arch = ... in`.

---

Resolved on most latest push!